sympy/solvers/pde.py

```diff
@@ -662,18 +677,20 @@ def pde_1st_linear_constant_coeff(eq, func, order, match, solvefun):
 def pde_1st_linear_variable_coeff(eq, func, order, match, solvefun):
     r"""
     Solves a first order linear partial differential equation
-    with variable coefficients. The general form of this partial differential equation is
+    with variable coefficients. The general form of this partial
+    differential equation is
 
     .. math:: a(x, y) \frac{df(x, y)}{dx} + a(x, y) \frac{df(x, y)}{dy}
                 + c(x, y) f(x, y) - G(x, y)
 
-    where `a(x, y)`, `b(x, y)`, `c(x, y)` and `G(x, y)` are arbitrary functions
-    in `x` and `y`. This PDE is converted into an ODE by making the following transformation.
+    where `a(x, y)`, `b(x, y)`, `c(x, y)` and `G(x, y)` are arbitrary
+    functions     in `x` and `y`. This PDE is converted into an ODE by
```

Comment:
Also remove unnecessary space after 'functions'.

Reply:
What do you mean by "unnecessary space after functions"?

Reply:
There is some extra space after the word 'functions' in the beginning of this line, that isn't needed.

Reply:
Oh, I see. I'll fix it

Reply:
Oh, I see. I'll fix it. Are we suppose to correct the source code to comply with PEP8?

Reply:
I am not sure if this is included in PEP8 or not. But since the current tests on travis fail because of this. That's why I suggested to change this.

Reply:
Ok, I think I made those changes.